src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -217,21 +227,27 @@ def make_suggestion_list(cls, config_file):
         ).suggest_migrations(config_file)
 
     @classmethod
-    def read_site_config(cls):
-        site_config_parser = ConfigParser()
-        init_site_config_parser(site_config_parser)
-        site_config_content = site_config_parser.parse(site_config_location())
-        return config_content_as_dict(site_config_content, {})
+    def read_site_config(cls, new_parser=USE_NEW_PARSER_BY_DEFAULT):
+        if new_parser:
+            return parse(site_config_location())
```

Comment:
This is a very very minor thing, but with the introduction of a new parser we now have multiple classes with `parse` - i.e `site_config_parser.parse(..)` . Would it be better - at least until we have removed the old ones - to have `lark_parser.parse( ... )` to avoid confusion?

Reply:
It's sort of difficult because we use a local import. We could rename the symbol to `lark_parse` easily if that works.

Reply:
Yea i know... And not very important - just felt `parse` drowns a bit in the different calls and could easily be mixed up ... But - i'm fine with it as is as well! And the PR seems promising - i'm looking forward to see it out "live" amongst all the configs out there 😅  Nice work of you two! 👍

Reply:
Changed it to `lark_parse` as that was the simplest solution :)